tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_private_navigation.py

```diff
@@ -16,12 +17,20 @@ def setUp(self):
         self.test_url = self.marionette.absolute_url('mozilla.html')
 
     def test_browser_private_navigation(self):
-        search = Search(self.marionette)
-        search.launch()
-        private_window = search.open_new_private_window()
+        self.device.hold_home_button()
+
+        cards_view = CardsView(self.marionette)
+        private_window = cards_view.open_new_private_window()
+
         browser = private_window.go_to_url(self.test_url)
         browser.switch_to_content()
```

Comment:
Strange, switch_to_content() was just added in this patch (in search/app.py), but there is no diff on this line.  maybe I'm looking at the wrong place?